Dockerfile

```diff
@@ -1,13 +1,13 @@
 ARG DOCKER_VERSION=19.03
-ARG PYTHON_VERSION=3.9.0
+ARG PYTHON_VERSION=3.7.9
```

Comment:
should we move back to 3.7.7 to avoid any surprise ? (that was in https://github.com/docker/compose/commit/a3e6e28eeb834435eca5a7e96a8016c7314c603e)

Reply:
We can do that if we see an issue but patch releases should be OK.